usr.sbin/daemon/daemon.c

```diff
@@ -297,8 +298,8 @@ main(int argc, char *argv[])
 	supervision_enabled = pidfile  != NULL ||
 		ppidfile != NULL ||
 		restart  != 0    ||
-		outfd    != -1   ||
-		syslog_enabled == true;
+		logpar.outfd    != -1   ||
+		logpar. syslog_enabled == true;
```

Comment:
why is this `syslog_enabled` not in `logpar`?

Reply:
small typo, I fixed it in one of the following commits

Reply:
fixed